pkg/kubelet/kuberuntime/util/util.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
nit: Let's use a specific pkg name if possible.

Reply:
given the import path is prefixed (i.e. consumers specify it as `runtimeutil "k8s.io/kubernetes/pkg/kubelet/kuberuntime/util"`), do you think it's still a problem?